looper/models.py

```diff
@@ -256,37 +375,90 @@ def __repr__(self):
         return "Project '%s'" % name + "\nConfig: " + str(self.config)
 
 
+    def finalize_pipelines_directory(self, pipe_path=""):
```

Comment:
Actually the vision should be that multiple directories can exist __simultaneously__ within the project and pipeline is chosen via this priority list at execution time; is this instead determining a single path at construction?